docs/topics/data-classes.md

```diff
@@ -53,11 +53,9 @@ data class Person(val name: String) {
 }
 ```
 
-In this example, only the `name` property can be used inside the `.toString()`, `.equals()`, `.hashCode()`, and `.copy()` implementations,
-and there is only one component function `.component1()`. The `age` property can't be used inside the `.toString()`, 
-`.equals()`, `.hashCode()`, and `.copy()` implementations because it's declared inside the class body. If two `Person` 
-objects have different ages but the same `name`, then they are treated as equal. This is because the `.equals()` function
-can only check for equality of the `name` property. For example:
+In this example, only the `name` property is used by default inside the `.toString()`, `.equals()`, `.hashCode()`, and `.copy()` implementations,
+and there is only one component function, `.component1()`. The `age` property, declared inside the class body, is not included in these automatically generated functions by default. 
```

Comment:
I feel that this passage could be even more straightforward, like "The `age` property is declared inside the class body and is excluded." Because we only need to come back to the statement at the beginning ("To exclude a property..."). And the following example makes it all clear, so let's get to it faster. What do you think?

Reply:
I agree. I made it a bit more concise in accordance with your comment. My original intention was to spell out everything in a very obvious way to ensure that we can clear up any confusion that originated the ticket. But I like the change - thank you. 👍  :)